{"name": "test_app_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestSuite' object has no attribute 'login'", "trace": "self = <scripts.test05_app_login.TestSuite object at 0x0000000003C28710>, phone = '13812345678', code = '246810'\n\n    def test_app_login(self, phone='13812345678', code='246810'):\n        # 调用app登录业务方法\n>       self.login.page_app_login(phone, code)\nE       AttributeError: 'TestSuite' object has no attribute 'login'\n\nscripts\\test05_app_login.py:35: AttributeError"}, "start": 1596636433367, "stop": 1596636433367, "uuid": "3a69ac7a-9e7f-4ad7-aba4-a06caf7c4a40", "historyId": "d32bbdd46c0c13171b1b742d3837ca68", "testCaseId": "bc361d1fdd7ceb4aac8ddb860da126fc", "fullName": "scripts.test05_app_login.TestSuite#test_app_login", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test05_app_login"}, {"name": "subSuite", "value": "TestSuite"}, {"name": "host", "value": "ETPUA4SBB4P3FBS"}, {"name": "thread", "value": "6904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test05_app_login"}]}